### DISCUSSION TITLE
FYI: updated the Community wiki page

### DISCUSSION THREAD

**Discussion by GerHobbelt** [6pac/SlickGrid#724] [Show and tell]
Augmented [that page](https://github.com/6pac/SlickGrid/wiki/The-SlickGrid-Community) with more info about my & other forks and the technology / features that made them SlickGrid too. 

Consider it a 'history' piece with an added description of the relevant remarkable SlickGrid features with a few hints about the technology choices made to get those done. 

Today, I think the **defer & slice** grid DOM renderer of the GerHobbelt fork still has merit and might be a nice addition for some folks using SlickGrid under particularly heavy machine loads, but it would (again) require some major effort to transfer it to the @6pac mainline.

The merged cells feature was & still is very nice too. Great if you need that sort of thing, e.g. when using the grid for more reporting-like grid layouts. *Definitely not my work* (I stole it from another fork 😉 ) and I haven't checked if [6pac](https://github.com/6pac/SlickGrid) already has it today (in one form or another). That would be another major haul/work to transfer, though.

Anyway. Enjoy!

**Comment by 6pac** [comment]
Thanks @GerHobbelt, you've always been one of the most active forks of the grid. Your comments bring a wealth of experience. We really need to have a sprint on our documentation soon at this repo!
